pyiron/interactive/scipy_minimizer.py

```diff
@@ -110,14 +123,42 @@ def run_static(self):
         else:
             self.status.not_converged = True
 
+    @staticmethod
+    def _tensor_to_voigt(s, strain=False):
```

Comment:
Can We use the same for Lammps `calc_md` and `calc_minimize`? - Does it make sense to have a `voigt` module or something like this?

Reply:
Yes probably it does, but I'd say that's a bit more than just a few extra lines that I would add in this PR because it sounds again like another Pandora's box...